Lib/inspect.py

```diff
@@ -819,7 +819,11 @@ def getfile(object):
     if ismodule(object):
         if getattr(object, '__file__', None):
             return object.__file__
-        raise TypeError('{!r} is a built-in module'.format(object))
+        if object.__spec__ is not None:
+            raise TypeError('{!r} is a built-in module'.format(object))
+        else:
+            raise TypeError('Custom module: {!r} is can not get source'
```

Comment:
```suggestion
            raise TypeError('Custom module: {!r} cannot get source'
```
Maybe this is better?

Reply:
yes and thanks